docs/architecture/deep-dives/passkeys/naming-convention.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 4
+---
+
+# Naming convention
+
+Since Bitwarden will both store passkeys in users' vaults and support log in with passkeys, it is
```

Comment:
Hmm, is `log in` used as a verb, subjective or noun? If verb `log in` is correct, else it should be `login`. Or is it a marketing term and we could make it italic.

Reply:
This should be "login".